client/src/components/body/profilepage/EnrolledGroups.js

```diff
@@ -34,67 +18,86 @@ const EnrolledGroups = (props) => {
     }
   }, [user, groupsUpdated]);
 
+  const displayEnrolledGroups = () => {
+    return enrolledGroups.map((group, index) => (
+      <EnrolledGroupCard
```

Comment:
![Screenshot 2024-10-25 at 1 17 11 PM](https://github.com/user-attachments/assets/91ba3135-a155-446b-b4f0-4747f8dd9855)
There needs to be some margin on the bottom